mail/chrome/messenger/aboutSupportMail.dtd

```diff
@@ -15,5 +15,6 @@
 <!ENTITY aboutSupport.accountsConnSecurity            "Bezpieczeństwo połączenia">
 <!ENTITY aboutSupport.accountsAuthMethod              "Metoda uwierzytelniania">
 <!ENTITY aboutSupport.accountsDefault                 "Domyślne?">
+<!ENTITY aboutSupport.identityName                    "Tożsamość">
```

Comment:
No introducing new ones is good enough for me but I don't have anything against separate patch removing them.